migrations/Version202402231420400684_qtiItemPci.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\qtiItemPci\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use Doctrine\Migrations\Exception\IrreversibleMigration;
+use oat\qtiItemPci\model\IMSPciModel;
+use oat\qtiItemPci\scripts\install\RegisterPciLikertScale;
+
+/**
+ * phpcs:disable Squiz.Classes.ValidClassName
+ */
+final class Version202402231520001465_qtiItemPci extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Update PCI LikertScaleInteraction (to IMS-compatible version)';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $registry = (new IMSPciModel())->getRegistry();
+        if ($registry->has('likertScaleInteraction')) {
+            /** @noinspection PhpUnhandledExceptionInspection */
+            $registry->removeAllVersions('likertScaleInteraction');
+        }
```

Comment:
**issue:** This should never be made, unless you perfectly know what you are doing..